frontend/packages/otr/src/components/clerkInterpreter/listing/ClerkInterpreterListingRow.tsx

```diff
@@ -21,29 +21,30 @@ export const ClerkInterpreterListingRow = ({
   const translateLanguage = useKoodistoLanguagesTranslation();
   const translateCommon = useCommonTranslation();
 
-  const dispatch = useAppDispatch();
-  const navigate = useNavigate();
-
-  const { lastName, nickName, qualifications } = interpreter;
+  const { id, lastName, nickName, qualifications } = interpreter;
 
   const visibleQualifications =
     QualificationUtils.getQualificationsVisibleInClerkHomePage(qualifications);
 
-  const handleRowClick = (e: React.MouseEvent<HTMLTableRowElement>) => {
-    e?.stopPropagation();
-    navigate(
-      AppRoutes.ClerkInterpreterOverviewPage.replace(
-        /:interpreterId$/,
-        `${interpreter.id}`
-      )
-    );
+  const overviewUrl = AppRoutes.ClerkInterpreterOverviewPage.replace(
+    /:interpreterId$/,
+    `${id}`
+  );
+
+  const dispatch = useAppDispatch();
+
+  const handleRowClick = () =>
     dispatch(setClerkInterpreterOverview(interpreter));
-  };
 
   return (
-    <TableRow className="cursor-pointer" onClick={handleRowClick}>
+    <TableRow
+      className="clerk-interpreter-listing__row"
+      onClick={handleRowClick}
```

Comment:
Ajetaanko handleRowClick enää ikinä?

Reply:
Pyryn kanssa juteltiin että tää feature tulee ehkä toteutettua toisella tavalla, avaan draftin vaihtoehtoisesta ratkaisusta. Jos mennään tällä niin pitää tsekata että tuo dispatch ajautuu / onClick tapahtuu.

Reply:
Ei tuota onClick-handleria pitäisi enää tarvita, otetaan se kokonaan pois jos mennään tällä mallilla.

Reply:
Tuon pitäisi musta estää se, ettei sivua avattaessa haeta tulkin tietoja backendiltä turhaan uudelleen. Kuinka tarpeellinen niin toinen juttu.

Reply:
No mutta tulkin tiedothan on ihan legitiimisti voinut päivittyä sen jälkeen kun listausnäkymässä ne tiedot on avattu. Mun mielestä toi on aika vaarallinen "optimointi".

Reply:
Hmmm... toki jos halutaan aina kääntäjän/tulkin tarkastelusivua avattaessa hakea ko. kääntäjän/tulkin tiedot backendiltä, niin silloin meidän sopimalla ajatusmallilla siitä, että listausrajapinnasta saa kaiken tarpeellisen tiedon yksittäisen entiteetin näyttämiseen ei ole enää perustetta. Sitä toki ei tarvitsisi tämän yhteydessä ruveta muokkaamaan, mutta suorituskykyoptimointina voisi myöhemmin halutessa listausrajapinnasta kaiken epäoleellisen datan poistaa.

Oon siis ok myös tämän kanssa että poistetaan täältä tämä onclick-handler mutta eikös käytännössä oltaisi silloin siinä tilassa, että listausrajapinnan osa datasta on turhaa? Lisäksi jos yksittäisen kääntäjän/tulkin hakurajapinnan kautta saisi kaiken datan (myös de-duplikoidut umpeutuneet auk./rek.), niin entiteetin muokkauksen jälkeen frontend voisi päivittää entiteetin tiedot listauksen stateen.

Reply:
No joo, totta tuokin. Ehkä ei tartte just nyt avata tuota keskustelua uudestaan.

Ehkä tuon onClick-handlerin olemassaolo häiritsi nyt enemmän kun kyseessä on ihan oikea linkki. Saa jättää noinkin.

Reply:
Jos mä sitten jätän tän näin. Mutta ehkä kommentin tästä vois tähän kohtaan koodia jättää miksi tämä `handleRowClick` on oleellinen. Ja voi sitten myöhemmin miettiä että mitä tälle haluaa tehdä jos tekee jotain.

Reply:
Mä tein tästä myös tiketin OPHOTRKEH-181. Voi miettiä haluaako tuota joskus ottaa työn alle vai ei.